src/components/Player.vue

```diff
@@ -97,13 +97,19 @@
         @click="updatePlayer('isVoteless', true)"
         title="Ghost vote"
       />
-
       <div
         class="name"
         @click="isMenuOpen = !isMenuOpen"
         :class="{ active: isMenuOpen }"
       >
-        {{ player.name }}
+        <font-awesome-icon
```

Comment:
I would always show the icon after the name, because usually people provide their pronouns after their name. (and yes, I realize that the tooltip will be on the left side for half the players)